admin/services/data-access/result.data.service.js

```diff
@@ -6,54 +6,16 @@ const TYPES = sqlService.TYPES
 const R = require('ramda')
 
 /**
- * Find pupils with scores
+ * Find pupils results data
  * @param {Number} schoolId
  * @param {Number} checkWindowId
  * @returns {Promise<*>}
  */
 resultDataService.sqlFindResultsBySchool = async (schoolId, checkWindowId) => {
```

Comment:
I wonder if this function should ensure that its arguments are present as they are required?

Reply:
@jon-shipley `result-service` `getPupilResultData ` method ensures these arguments are valid before calling this method. There should be no logic in the data layer as services should be assigned for these tasks.